Library/Formula/moc.rb

```diff
@@ -51,4 +52,61 @@ def caveats
             --with-ncurses
     EOS
   end
+
+  test do
+    assert_match /Version/, `#{bin}/mocp --version`
```

Comment:
That's certainly better, will use that one. To ease burden on the CI, I won't push changes until the patch is accepted into Homebrew/patches.